### PR TITLE
Google Cloud Marketplace pages

### PR DESCRIPTION
**Pull request by kevinbowman-aiven**
# What changed, and why it matters

Made the "set up" page for Google Cloud Marketplace more of a step-by-step instructions, and added a new "move to" page to describe how to transition from direct billing to Google Cloud Marketplace billing.

### CODE REVIEW

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 1]
```diff
@@ -1,32 +1,43 @@
-Set up Google Cloud Platform Marketplace for Aiven Services
-===========================================================
+Set up Google Cloud Marketplace for Aiven services
+==================================================
 
-Aiven makes its services available through the Google Cloud Platform Marketplace. This article shows the steps needed to create a subscription that links the accounts.
+Aiven makes its services available through the Google Cloud Marketplace on GCP. This article shows the steps needed to create a subscription that links the accounts.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:4]:
```suggestion
Aiven makes its services available through the Google Cloud Marketplace on Google Cloud Platform (GCP). This article shows the steps needed to create a subscription that links the accounts.
```

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
 
-Navigate to Aiven marketplace page on GCP
------------------------------------------
+First, there are some steps which need to be completed on the Google Cloud Marketplace page, before heading over to the Aiven console and finishing the process.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:6]:
```suggestion
First, there are some steps that need to be completed on the Google Cloud Marketplace page before heading over to the Aiven console and finishing the process.
```

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
 
-Navigate to `Aiven on the GCP marketplace <https://console.cloud.google.com/marketplace/product/aiven-public/aiven>`_. This page contains information about all of Aiven's services and how the marketplace subscription works.
+Google Cloud Marketplace setup
+------------------------------
 
-Select your desired billing account, then read and agree to the the
-terms and conditions.
+1. Navigate to `Aiven Managed Database Services on the Google Cloud Marketplace <https://console.cloud.google.com/marketplace/product/aiven-public/aiven>`_.  This page contains information about all of Aiven's services and how the marketplace subscription works.  Click the "Subscribe" button on this page.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:11]:
```suggestion
1. Navigate to `Aiven Managed Database Services on the Google Cloud Marketplace <https://console.cloud.google.com/marketplace/product/aiven-public/aiven>`_.  This page contains information about all of Aiven's services and how the marketplace subscription works.  Click the **Subscribe** button on this page.
```

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
-after deploying Aiven services.
+2. Select your desired billing account, then read and agree to the terms and conditions.
 
-Finalize your Aiven GCP subscription
-------------------------------------
+3. When you are ready, click the **Subscribe** button at the bottom of the page.  You will NOT be charged by clicking this button; this only sets up a billing subscription between GCP and Aiven.  You will only be charged after deploying Aiven services.
 
-After subscribing, check your email for a confirmation link to to create your
-new account. Click the link to confirm this.
+4. You should now see a message "Your order request has been sent to Aiven".  Click on the "Go to product page" button.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:17]:
```suggestion
4. You should now see a message that says  "Your order request has been sent to Aiven".  Click on the **Go to product page** button.
```

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 5]
```diff
@@ -1,32 +1,43 @@
...
 
-When you view the Aiven page on GCP Marketplace, you will see a new button labeled **MANAGE ON PROVIDER**. Clicking this button will take you to the Aiven GCP Console at https://console.gcp.aiven.io.
+5. Everything is now complete in your GCP account, but you still need to setup the Aiven account.  Click on the "Manage on provider" button to go to the Aiven console to complete the process.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:19]:
```suggestion
5. Everything is now complete in your GCP account, but you still need to setup the Aiven account.  Click on the **Manage on provider** button to go to the Aiven console to complete the process.
```

File: docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst [hunk 6]
```diff
@@ -1,32 +1,43 @@
...
+6. You should now be on a signup page at Aiven, asking you for your email address to create a new account.
+
+7. After entering your email address, you will be sent an email to confirm your registration.  Click on the link.
+
+8. You can now proceed to the `Aiven console for GCP <https://console.gcp.aiven.io/>`_, where you can manage your Aiven services as normal.
+
+.. image:: /images/platform/howto/gcp-console.png
+   :alt: The GCP version of the Aiven web console
 .. note:: 
+   Note the URL is https://console.gcp.aiven.io - this uses a different account system than https://console.aiven.io.  If you have an existing Aiven account you will need to create a new Aiven GCP account using the Aiven GCP console, and when coming back to Aiven in the future you will need to use https://console.gcp.aiven.io to login.
```
Comment (dewan-ahmed) [inline docs/platform/howto/billing-google-cloud-platform-marketplace-subscription.rst:37]:
```suggestion
   Note the URL is https://console.gcp.aiven.io - this uses a different account system than https://console.aiven.io.  If you have an existing Aiven account you will need to create a new Aiven GCP account using the Aiven GCP console, and when coming back to Aiven in the future, you will need to use https://console.gcp.aiven.io to login.
```

File: docs/platform/howto/move-to-gcp-marketplace-billing.rst [hunk 7]
```diff
@@ -0,0 +1,20 @@
+Move from Aiven direct billing to Google Cloud Platform Marketplace
+===================================================================
+
+Aiven makes its services available through the Google Cloud Platform Marketplace.  If you already have some services running in a project which is billed directly through Aiven, but you would like to move to a Google Cloud Platform Marketplace subscription without disrupting your services, this article shows the steps needed to gather the relevant information and submit the request.
```
Comment (dewan-ahmed) [inline docs/platform/howto/move-to-gcp-marketplace-billing.rst:4]:
```suggestion
Aiven makes its services available through the Google Cloud Platform Marketplace.  If you already have some services running in a project which is billed directly through Aiven but you would like to move to a Google Cloud Platform Marketplace subscription without disrupting your services, this article shows the steps needed to gather the relevant information and submit the request.
```

File: docs/platform/howto/move-to-gcp-marketplace-billing.rst [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+Create a new account using a Google Cloud Platform Marketplace subscription
+---------------------------------------------------------------------------
+
+Follow the steps to :doc:`set up Google Cloud Platform Marketplace for Aiven Services <billing-google-cloud-platform-marketplace-subscription>`.  This will create a new empty project, which is where your services will be moved to.
+
+Gather the required information
+-------------------------------
+
+Aiven will need the name of the project which currently contains your services, and the account name for the new project (in the format `E-1234-1234-1234-1234`) which is shown in the "Settings" screen for your new project in the Aiven GCP console (`https://console.gcp.aiven.io <https://console.gcp.aiven.io>`_)
```
Comment (dewan-ahmed) [inline docs/platform/howto/move-to-gcp-marketplace-billing.rst:14]:
```suggestion
Aiven will need the name of the project which currently contains your services and the account name for the new project (in the format `E-1234-1234-1234-1234`) which is shown in the "Settings" screen for your new project in the Aiven GCP console (`https://console.gcp.aiven.io <https://console.gcp.aiven.io>`_).
```

**Review by dewan-ahmed — COMMENTED** [review]
Hi @kevinbowman-aiven thanks for this contribution. I added minor styling feedback. 

For billing-google-cloud-platform-marketplace-subscription.html#billing section, I couldn't locate **container_name** and **resource_name** from the image below that section. 

Thanks.

**Comment by kevinbowman-aiven** [comment]
Thanks @dewan-ahmed, I'll take a look.  I didn't change that billing section from the previous version of the content, but I'll verify that it's still accurate (or change it accordingly).

**Comment by kevinbowman-aiven** [comment]
@dewan-ahmed I'm going to have to wait for a billing cycle before the data flows to confirm that screenshot.  As I haven't changed it from the previous version, I suggest proceeding with the other changes and I'll make a note to re-visit this when the data is in the right place for me to check it?  If any changes are needed at that point then I'll raise a separate PR.

**Pull request merged**